### ISSUE TITLE
multiview no longer prints to scale

### ISSUE THREAD

**Issue by orthocast** [kborer/maxilla#1]
When the Multi-view box is selected the print button is supposed to send a full sized 1:1 image to the printer as earlier versions did. They are coming out slightly undersized. If the Save as PDF button is clicked and that image is printed it works correctly.

It’s really not easily visible. You have to measure the print with a ruler and compare that with the actual models or take a point to point measurement with the viewer. To do this you must download the viewer and try a sample model file. 

![maxilla viewer](https://user-images.githubusercontent.com/42647584/44533625-f3910080-a6c3-11e8-9be6-0f40a3813265.jpg)
![printed image](https://user-images.githubusercontent.com/42647584/44533626-f3910080-a6c3-11e8-876d-b0c7da593570.jpg)

[Carroll.pdf](https://github.com/kborer/maxilla/files/2315029/Carroll.pdf)
[Li.pdf](https://github.com/kborer/maxilla/files/2315030/Li.pdf)

![printed image](https://user-images.githubusercontent.com/42647584/44533639-f986e180-a6c3-11e8-86ef-f57cd018641f.jpg)

[Carroll.pdf](https://github.com/kborer/maxilla/files/2315032/Carroll.pdf)
[Li.pdf](https://github.com/kborer/maxilla/files/2315033/Li.pdf)

![maxilla viewer](https://user-images.githubusercontent.com/42647584/44533641-fa1f7800-a6c3-11e8-8abf-7aa440477ba6.jpg)

**Comment by jlodos** [comment]
Hi
I believe I fixed this in the attached (debug) version. If the fix is correct the "Save PDF" option generated slightly undersized images as well.
I could send a PR for the sources. Please contact me at jlodos at yahoo.com. 
[maxilla.zip](https://github.com/kborer/maxilla/files/2363741/maxilla.zip)

**Comment by orthocast** [comment]
I get this error message when i try to install the fix. How do i proceed? 

![image](https://user-images.githubusercontent.com/42647584/45267575-100b9780-b43d-11e8-980c-8d7b3c233981.png)
Windows 7professional

**Comment by jlodos** [comment]
One possibility is to install the old version with the existing installer and replace the executable in the installation folder. I recommend this.

Alternatively if you find another missing dll please let me know and I will send it. I believe this is more work for you.
Here is glut32.dll debug. I modified all projects to compile with VS2017 and modern C++ but in principle external dlls should work the same as before.
[glut32.zip](https://github.com/kborer/maxilla/files/2364538/glut32.zip)

**Comment by orthocast** [comment]
Jlodos,

I installed the exe file as you recommended and when I print “the Multiview” from the print button it still prints undersized compared to printing from the save PDF button. Clicking the print button creats a PDF as well but it is not to scale as it should be. I attached some examples showing how to measure with the app and from the print using a ruler.

Sincerely,

Gordon Rigby
800-635-5663
Ortho Cast, Inc.
99 N Main St
High Bridge, NJ 08829
